test/test_nngraph.lua

```diff
@@ -343,7 +343,7 @@ function test.test_annotateGraph()
   local function checkDotFile(tmpfile)
     local dotcontent = io.open(tmpfile .. '.dot', 'r'):read("*all")
     tester:assert(
-        dotcontent:match('%[label=%"Input.*DescA.*%" color=red%]'))
+        dotcontent:match('%[label=%"Input.*DescA.*%" color=red.*%]'))
```

Comment:
this seems unrelated to this PR, maybe minor point, but it would be better to have it separately.